connect-publish/src/connect-publish.ts

```diff
@@ -76,7 +77,9 @@ export async function connectPublish (args: ActionArgs): Promise<ConnectPublishR
   const client = new rsconnect.APIClient({ apiKey: args.apiKey, baseURL })
   await client.serverSettings()
 
-  const { accessType, dirs, force, ns, requireVanityPath, showLogs, updateEnv } = args
+  const { accessType, dirs, force, ns, requireVanityPath, showLogs, updateEnv, workingDirectory } = args
+
+  process.chdir(workingDirectory)
```

Comment:
This is process-level adjustment, which would adjust the working directory for other, asynchronous code.

It looks like this is never the case, since the only caller is the top-level `run()`.

https://github.com/rstudio/actions/blob/80855bdf2b0cae15eebf2f0dde2529ee91a769a0/connect-publish/src/main.ts#L4-L6

Could you confirm?

Reply:
Is it worth adding tests confirming that `dir` is resolved according to `working-dir` rather than the initial working directory?

Reply:
The testing situation is not wonderful at the moment 😅 Help very much wanted!

Reply:
Regarding `chdir` and process-level / async, this whole thing gets run as a script with the expectation that only one `connectPublish` is happening at a time per-process, so I think this is safe enough 👍🏼